Makefile

```diff
@@ -170,6 +170,15 @@ list-inventory: ansible/inventory-list.json
 	cd ansible &&\
 	cat inventory-list.json | ./dump_inventory.py
 
+provision: check-deploy-env
```

Comment:
Could you please move it up in the file where all other playbooks are set?

What happens if the BOOTSTRAP_VERSION is not set?

Reply:
defaults to what's in playbook, e.g. `stable` https://github.com/aeternity/infrastructure/blob/b093dd58017b5598daf94e32c96a6edb00cb2cee/ansible/async_provision.yml#L8